imagetest/test_suites/metadata/setup.go

```diff
@@ -1,17 +1,87 @@
 package metadata
 
 import (
+	"fmt"
+	"math/rand"
+
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
 )
 
 // Name is the name of the test package. It must match the directory name.
 var Name = "metadata"
 
+const (
+	shutdownScript = `#!/bin/bash
+
+while [[ 1 ]]; do
+  date +%s >> /shutdown.txt
+  sync
+  sleep 1
+done`
+	shutdownScriptTemplate = `#!/bin/bash
+echo "%s" > %s`
+	shutdownContent    = "The shutdown script worked."
+	shutdownOutputPath = "/shutdown_out.txt"
+	shutdownMaxLength  = 32768 // max shutdown metadata value
+	shutdownScriptURL  = "gs://gcp-guest-cloud-test-artifacts/shutdown-script.txt"
```

Comment:
this makes the test only work for us. why do we need a test artifact to pre-exist? we can accomplish the same thing by writing the content to a file that will get uploaded by the daisy workflow.

Reply:
Good catch, wee could config this in setup.